src/components/navigation/Link.tsx

```diff
@@ -23,35 +28,45 @@ const Link = function Link({
   elementRef,
 
   underline = 'none',
-  color = 'brand',
+  color,
+  unstyled = false,
+  variant = 'brand',
 
   ...htmlAttributes
 }: LinkProps) {
+  // Map from deprecated `color` prop to `variant` if `color` is present
+  const theme = typeof color === 'string' ? color : variant;
+  const styled = !unstyled;
+  const themed = styled && theme !== 'custom';
+
   return (
-    <LinkBase
+    <a
       data-component="Link"
+      rel="noopener noreferrer"
       {...htmlAttributes}
-      classes={classnames(
-        'rounded-sm',
-        // NB: Base classes are applied by LinkBase
+      className={classnames(
         {
-          // color
-          'text-brand hover:text-brand-dark': color === 'brand', // default
-          'text-color-text-light hover:text-brand': color === 'text-light',
-          'text-color-text hover:text-brand-dark': color === 'text',
+          'focus-visible-ring rounded-sm': styled,
+          // underline
+          // TODO: Underline should be controlled by `variant` and should default
+          // to `always`
+          'no-underline hover:no-underline': styled && underline === 'none', // default
+          'underline hover:underline': styled && underline === 'always',
+          'no-underline hover:underline': styled && underline === 'hover',
         },
```

Comment:
Since `styled` is a condition for each one of the styles, perhaps we could consider a different formula.

```tsx
className={classnames(
  {
    'focus-visible-ring rounded-sm': styled,
  },
  styled && {
    'no-underline hover:no-underline': underline === 'none', // default
    'underline hover:underline': underline === 'always',
    'no-underline hover:underline': underline === 'hover',
  },
  // [...]
)}
```

It would allow not having to add the condition every time, but perhaps it makes is less clear, or too subtle. What do you think?